tests/test_api.py

```diff
@@ -35,7 +37,7 @@ def test_get_current_span_default_hub(sentry_init):
 
     assert get_current_span() is None
 
-    scope = Scope.get_current_scope()
+    scope = get_current_scope()
```

Comment:
I would like it more to always call `sentry_sdk.get_x()` to make it more explicit. But your call.

Reply:
this follows the structure in the rest of the file since this is testing the api